Bottom: 747d4554de2c9979ee7644ccff0d9921444125d5
Top:    fd4d905c3efcf81f80b22150a19fb5630881d10e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-10 23:24:15 -0500

hid/gtk: Fix stupid (missing semicolon) bug breaking compile

I was _sure_ I'd buid tested this before pusing... obviously not!


---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index b69a8e4..7e28028 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -825,7 +825,7 @@ ghid_layer_selector_get_accel_group (GHidLayerSelector *ls)
 struct layer_data {
   GHidLayerSelector *ls;
   gint user_id;
-}
+};
 
 /*! \brief used internally */
 static gboolean
